app/uk/gov/hmrc/eoricommoncomponent/frontend/views/date_of_establishment.scala.html

```diff
@@ -37,7 +37,6 @@
 
 
 @introAndHintHtml = {
-    <p id="date-of-establishment-label" class="govuk-body">@Html(DateOfEstablishmentViewModel.introText(orgType))</p>
     <label id="date-of-establishment-hint-text" class="govuk-hint">@Html(messages("cds.date-established.hint"))</label>
 }
```

Comment:
This doesn't need to be a label - what we can actually just do is add the below as the param to inputDate_di:
```
hintText = Some(Text(messages("cds.date-established.hint")))